mtp_api/apps/prison/tests/test_views.py

```diff
@@ -26,12 +28,35 @@ def setUp(self):
     def list_url(self):
         return reverse('prisonerlocation-list')
 
-    def test_fails_without_permissions(self):
-        unauthorised_user = self.prison_clerks[0]
+    def test_fails_without_application_permissions(self):
+        """
+        Tests that if the user logs in via a different application,
+        they won't be able to access the API.
+        """
+        for unauthorised_user in [
+            self.prison_clerks[0], self.bank_admins[0]
```

Comment:
Good point, it might be worth adding. Not sure if I can make it elegant enough but I'll do my best